.azure-pipelines/win32-conda-CI.yml

```diff
@@ -34,6 +36,7 @@ jobs:
       conda install -c conda-forge protobuf
       conda install -c conda-forge numpy
       python -m pip install --upgrade pip
+      test '$(python.version)' == '3.7' && pip install https://github.com/xadupre/onnx/releases/download/1.3.0/onnx-1.3.0-cp37-cp37m-win_amd64.whl
```

Comment:
Can you check in the onnx package fixing for python 3.7? I heard they got some trouble on python 3.7.

Reply:
I'll check. I'm surprised. I've been working with 3.7 since last september.